tests/datasets/test_benin_cashews.py

```diff
@@ -16,15 +16,15 @@
 from torchgeo.datasets import BeninSmallHolderCashews
 
 
-class Dataset:
+class Collection:
     def download(self, output_dir: str, **kwargs: str) -> None:
```

Comment:
When radiant-mlhub 0.5 was first released, we actually didn't notice when things broke because we replace their download method with our own. Do you know if it's possible to monkeypatch something else farther internal so that we can actually test their download method but still use a local file?

Reply:
I actually have never used monkeypatch! So I'm not sure how that would work. It looks like the `download` call just creates a `requests. Session` and download chunks for the dataset in a threadpool. We would probably have to over-ride something in requests? But then our test data would need to be chunked I think.